.github/workflows/push.yml

```diff
@@ -41,3 +41,10 @@ jobs:
         run: |
           docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG .
           docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
+      - name: Copy otp jar artifact to S3 bucket
+        env:
+          JAR_FILE: otp_${{ github.sha }}.jar
+          BUCKET_NAME: s3://${{ S3_BUCKET_NAME }}
```

Comment:
z ciekawości. Gdzie te zmienne są ustawione?

Reply:
This is added as a repository secret, there should be `secrets.S3_BUCKET_NAME`. Fixed.